src/ddmd/dsymbolsem.d

```diff
@@ -4744,9 +4744,9 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
             if (funcdecl.isOverride)
             {
                 if (funcdecl.storage_class & STCdisable)
-                    funcdecl.deprecation("overridden functions cannot be annotated @disable");
+                    deprecation(funcdecl.loc, "overriding function %s cannot be annotated @disable", funcdecl.toPrettyChars);
```

Comment:
I think these messages should use a different tense:  "overridden function..." instead of "overriding function...".

Reply:
Also, surround %s with tick marks (`).

Reply:
Furthermore, the problem described in the issue is not about the `@disable` attribute, but deprecation. I think the error message should instead be "overridden function ``%s`` cannot be deprecated", but a more general "overridden functions cannot be deprecated" may be even better.

Reply:
I am not a native English speaker but isn't overridden used for the function that is being overridden and overriding for the one that overrides?

> Furthermore, the problem described in the issue is not about the @disable attribute, but deprecation

Yes, it solves that one too. But the error regarding disabled was also wrong, so I tried to solve that one too.

> I think the error message should instead be "overridden function %s cannot be deprecated"

That's what it says except that I used overriding.

Reply:
> ...isn't overridden used for the function that is being overridden and overriding for the one that overrides?

Ah, I see what you meant. In that case it should say "The overriding function ``%s`` cannot be ...".  Without the "The", you're telling the user **that the action of** overriding function xx cannot be annotated with @disable, which is a weird thing to say.  Actually, it can be interpreted either way, so at a minimum, prepend the article to disambiguate.

> Yes, it solves that one too. But the error regarding disabled was also wrong, so I tried to solve that one too.

Ok, I understand.  

These would be correct:
"the overriding function ``%s`` cannot be annotated with ``@disable``"
"the overriding function ``%s`` cannot be marked as deprecated"
It still sounds a bit weird due to the overloaded uses of "override" in the English language.

I would prefer something like:
"``%s`` cannot be annotated with ``@disable`` because it is overriding a function in the base class"
"``%s`` cannot be marked as deprecated because it is overriding a function in the base class"

I am a native English speaker, but that doesn't make me an authority on the subject.

Reply:
I see. Thanks for the explanation.

> I would prefer something like:
"%s cannot be annotated with @disable because it is overriding a function in the base class"
"%s cannot be marked as deprecated because it is overriding a function in the base class"

Indeed it does sound better that way.

Reply:
Done. Thanks!